clients/src/main/java/org/apache/kafka/common/requests/FetchRequest.java

```diff
@@ -228,12 +240,18 @@ public FetchRequest build(short version) {
             }
 
             FetchRequestData fetchRequestData = new FetchRequestData();
-            fetchRequestData.setReplicaId(replicaId);
             fetchRequestData.setMaxWaitMs(maxWait);
             fetchRequestData.setMinBytes(minBytes);
             fetchRequestData.setMaxBytes(maxBytes);
             fetchRequestData.setIsolationLevel(isolationLevel.id());
             fetchRequestData.setForgottenTopicsData(new ArrayList<>());
+            if (version < 15) {
+                fetchRequestData.setReplicaId(replicaId);
+            } else {
+                fetchRequestData.setReplicaState(new ReplicaState()
+                        .setReplicaId(replicaId)
+                        .setReplicaEpoch(replicaEpoch));
```

Comment:
nit: Could we use 4 spaces to indent this two lines to be consistent with similar code in this method?